docs/user-guide/installandconfig.md

```diff
@@ -35,7 +35,13 @@ The following information is required during the installation process. Software
    
      In order for above started tasks to run correctly, security manager configuration needs to be performed.  This is documented in [Configuring the z/OS system for Zowe](configure-zos-system.md) and a sample JCL member `ZWESECUR` is shipped with Zowe that contains commands for RACF, TopSecret, and ACF2 security managers.  
 
-**Note:** To start the API Mediation Layer as a standalone component, see [API Mediation Layer as a standalone component](api-mediation-standalone.md).
+  **Notes:**
+  
+  - To start the API Mediation Layer as a standalone component, see [API Mediation Layer as a standalone component](api-mediation-standalone.md).
+  
+  - If you plan to use API ML with basic authentication and JSON web token authentication, you need to run only `ZWESVSTC`. No need to run `ZWESISTC` and `ZWESASTC`.
+  
+  - If you plan to use API ML with x509 client-side certificate authentication, you need to run `ZWESISTC`.
```

Comment:
What about ZWESVSTC? Your current wording implies it is not required.

Reply:
Not sure what you mean by the 'current wording'. Do you refer to line 42?

Reply:
the comment was about
 If you plan to use API ML with x509 client-side certificate authentication, you need to run `ZWESISTC`.

Reply:
Maybe it could be worded you need to run `ZWESVSTC` and `ZWESISTC`.